demes/demes.py

```diff
@@ -1143,6 +1143,36 @@ def time_span(self):
         """
         return self.start_time - self.end_time
 
+    def size_at(self, time: float) -> float:
+        """
+        Get the size of the deme at a given time.
+
+        :param float time: The time at which the size should be calculated.
+        :return: The deme size.
+        :rtype: float
+        """
+        for epoch in self.epochs:
+            if epoch.start_time >= time >= epoch.end_time:
```

Comment:
Maybe it's the sleep deprivation, but I can't get my head around this condition - shouldn't it be ``epoch.start_time <= time < epoch.end_time``? As in, we break if the time is within this epoch's interval?

Reply:
Two things: (1) `start_time > end_time`, so we must have `start_time > time >= end_time`. And (2) we also want to accept `time=math.inf`, so we check `start_time >= time >= end_time`. Given this is in a loop, and the epochs are time-sorted, the start_time = time condition is only hit for the first epoch, and only if the time is infinity.

Reply:
Ack, I'm thinking backwards in time as usual, thanks, my bad!